examples/elastic/src/main/java/com/hazelcast/jet/examples/elastic/ElasticSinkExample.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.examples.elastic;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.hazelcast.jet.Jet;
+import com.hazelcast.jet.JetInstance;
+import com.hazelcast.jet.core.processor.SourceProcessors;
+import com.hazelcast.jet.elastic.ElasticSinks;
+import com.hazelcast.jet.pipeline.BatchSource;
+import com.hazelcast.jet.pipeline.Pipeline;
+import org.elasticsearch.action.index.IndexRequest;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static com.hazelcast.jet.pipeline.Pipeline.create;
+import static com.hazelcast.jet.pipeline.Sources.batchFromProcessor;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class ElasticSinkExample {
+
+    public static void main(String[] args) {
+        try {
+            ObjectMapper mapper = new ObjectMapper();
```

Comment:
Yes, it is thread-safe (with the exception that all ObjectMapper's configuration must be done before first use, but there the defaults are used).

I am not sure. ObjectMapper is serializable, so it can be used with map:
```
ObjectMapper mapper = new ObjectMapper();
...
 .map(json -> mapper.readValue(json, new TypeReference<Map<String, Object>>() {})
```
vs
```
ServiceFactory<?, ObjectMapper> service = ServiceFactories.sharedService((context) -> new ObjectMapper());
...
.mapUsingService(service, (mapper, json) -> mapper.readValue(json, new TypeReference<Map<String, Object>>() {}))
```

The one bit I don't like here is the `new TypeReference<Map<String, Object>>() {})`, which we could cache inside a service. But then the service wouldn't be just the ObjectMapper, which would complicate the example